### PR TITLE
spec: add rendering information

### PR DESCRIPTION
**Pull request by joshmoore**
This formalizes the output of webgateway/imgData for the moment
as consumed by ome-zarr-py.

cc: @manics @will-moore @mtbc @jburel

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -19,10 +20,72 @@ for public re-use.
...
+"channels": [                         # Array matching the c dimension size
+    {
+        "active": true,
+        "coefficient": 1,
+        "color": "0000FF",
+        "emissionWave": null,
+        "family": "linear",
+        "inverted": false,
+        "label": "LaminB1",
+        "reverseIntensity": false,
```
Comment (will-moore) [inline spec.md:66]: I think we can remove this - it is deprecated to use 'inverted' instead.

File: spec.md [hunk 2]
```diff
@@ -19,10 +20,72 @@ for public re-use.
...
+
+```
+"id": 1,                              # ID in OMERO
+"name": "example.tif",                # Name as shown in the UI
+"channels": [                         # Array matching the c dimension size
+    {
+        "active": true,
+        "coefficient": 1,
+        "color": "0000FF",
+        "emissionWave": null,
```
Comment (will-moore) [inline spec.md:62]: Don't think we need emissionWave - it was used for channel label but is now redundant.

Comment (joshmoore) [reply spec.md]: Done (on minio-dev as well)

**Comment by joshmoore** [comment]
Got a general :thumbsup:

**Pull request merged**